salt/modules/boto_rds.py

```diff
@@ -197,6 +197,30 @@ def parameter_group_exists(name, tags=None, region=None, key=None, keyid=None,
         return resp
 
 
+def db_cluster_exists(name, tags=None, region=None, key=None, keyid=None,
+                           profile=None):
+    '''
+    Check to see if an RDS db cluster exists.
+
+      .. versionadded:: Fluorine
+    CLI example::
```

Comment:
Can you change this to:
```
CLI Example:

.. code-block: bash
```
Instead of `CLI example::`? Our docs have some handy formatting that looks for the code-block type of CLI examples.

Reply:
This seems reasonable but it's worth noting that what's in our PR is consistent with every other block inside this particular file (which granted, is probably way out of date).